kit/ChildSession.cpp

```diff
@@ -832,6 +859,50 @@ bool ChildSession::getStatus(const char* /*buffer*/, int /*length*/)
     return sendTextFrame("status: " + status);
 }
 
+void ChildSession::dumpRecordedUnoCommands()
+{
+    std::atomic<char*>* recordedCommands = unoCommandsRecorder.getRecordedCommands();
+
+    for(int i = 0; i < unoCommandsRecorder.NUM_UNO_COMMANDS; i++)
```

Comment:
At least here is a space missing. Sadly I am not sure if our git hooks try to enforce use of clang-format or not.

Reply:
In libreoffice core they did. Now I dont know what is happening. But the hooks are installed and working because I am getting error messages of trailing whitespaces. Also code signing is done by hooks so...